cluster/gce/configure-vm.sh

```diff
@@ -260,6 +260,13 @@ function create-salt-auth() {
   kubelet_auth_file="/srv/salt-overlay/salt/kubelet/kubernetes_auth"
   (umask 077;
     echo "{\"BearerToken\": \"${KUBELET_TOKEN}\", \"Insecure\": true }" > "${kubelet_auth_file}")
+
+  # Generate tokens for other "service accounts".  Append to known_tokens.
+  for a in scheduler controller_manager logging monitoring dns
+  do
+    token=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+    echo "${token},kubernetes_builtin:$a,kubernetes_builtin:$a" >> "${known_tokens_file}"
```

Comment:
We ended up with normal usernames being unnamedspaced (so they can inherit the preferred username from the bedrock identity provider), and namespaced system/service account usernames with a "prefix:" (like "system:" or "project:"). That means ":" is disallowed in normal usernames for us.